docs/user/agent/mce-integration.md

```diff
@@ -3,6 +3,135 @@
 This document describes how to install the multicluster engine for Kubernetes operator (mce) and how to deploy the cluster zero (hub) using the agent-based installer for Openshift.
 The procedure is partially automated, and it will require some manual steps after the initial cluster deployment.
 
+## Installing while disconnected from the greater internet
+
+### Environment Pre-requisites
+
+[Create a local image registry in your environment.](https://docs.openshift.com/container-platform/4.11/installing/disconnected_install/index.html) Make note of the DNS name and port of your local registry.
+
+Download the [oc mirror](https://github.com/openshift/oc-mirror) tool.
+
+#### Agent Installer pre-requisites
+
+For the complete pre-requisites description, please check section 1.2.1 in [[1]](#1).
+
+Create a folder (`<asset dir>`) and place a valid `install-config.yaml` and `agent-config.yaml`.
+
+### Mirror the Openshift release
+
+Using the oc mirror tool mirror the desired Openshift release (4.11+). Here is an example ImageSetConfiguration for OCP 4.11:
+
+__ocp-imageset.yaml__
+```
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+archiveSize: 4
+storageConfig:
+  registry:
+    imageURL: <your-local-registry-dns-name>:<your-local-registry-port>/origin:latest
+    skipTLS: true
+mirror:
+  platform:
+    channels:
+    - name: candidate-4.11
+      type: ocp
+  additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest
+```
+
+With this file you will be able to use this command to mirror the OCP release:
+
+```oc mirror --dest-skip-tls --config ocp-imageset.yaml docker://<your-local-registry-dns-name>:<your-local-registry-port>```
+
+### Mirror the MCE and LSO Operators
+
+Mirror the `multicluster-engine` (2.0.2+) and `local-storage-operator` operators to your environment. Here is an example ImageSetConfiguration for these operators.
+
+__mce-lso-imageset.yaml__
+```
+---
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+mirror:
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+      packages:
+        - name: multicluster-engine
+---
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+mirror:
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+      packages:
+        - name: local-storage-operator
+```
+
+Use the following command to mirror these two operators to your environment.
+
+```oc mirror --dest-skip-tls --config mce-lso-imageset.yaml docker://<your-local-registry-dns-name>:<your-local-registry-port>```
+
+### Update the mirror configurations
+
+Create a folder in the `<asset dir>` called `mirror`.
+
+Update the following `registries.conf` file with your local registry information. Copy this file in to `<asset dir>/mirror` folder.
+
+__registries.conf__
```

Comment:
Q: Since one of the pre-req is to use the `install-config.yaml` file, would be possible to specify the `ImageContentSource` (and also `additionalTrustBundle`) field in the `install-config.yaml` instead of providing directly the `registries.conf` (given the `registries.conf` will be generated from that field)? Probably it could be easier for the user

Reply:
🤔 Yes I can update it to provide those sections of the `install-config.yaml` as that would be better for the customer.